packages/backend/src/payment-method/ilp/connector/core/rafiki.ts

```diff
@@ -210,14 +187,6 @@ export class Rafiki<T = any> {
     return response.rawReply
   }
 
-  //public get router(): Router | undefined {
```

Comment:
Thanks for removing these, I think there are just a few of these left in `/src/payment-method/ilp/connector/core/rafiki.ts`

Reply:
Found some more, updated. 💪